pyogrio/tests/test_core.py

```diff
@@ -451,8 +451,9 @@ def test_read_info_force_total_bounds(
         assert info["total_bounds"] is None
 
 
+@pytest.mark.filterwarnings("ignore: Measured")
 def test_read_info_without_geometry(test_fgdb_vsi):
-    assert read_info(test_fgdb_vsi)["total_bounds"] is None
+    assert read_info(test_fgdb_vsi, layer="basetable_2")["total_bounds"] is None
```

Comment:
What warning does this give right now?

Reply:
It looks like some changes related to #362 got mixed up in this PR, not sure how that happened. I rolled back the changes that belong in #362.